cms/djangoapps/contentstore/views/course.py

```diff
@@ -771,8 +878,54 @@ def course_index(request, course_key):
         # gather any errors in the currently stored proctoring settings.
         advanced_dict = CourseMetadata.fetch(course_module)
         proctoring_errors = CourseMetadata.validate_proctoring_settings(course_module, advanced_dict, request.user)
+        chatly_widget = ChatlyWidget.objects.filter(course_key=course_key)      
+        chatly_data = get_course_delta(course_structure, chatly_widget)
+        widget_data = {
+            'prompt':escapejs(DEFAULT_QA_PROMPT),
+            'temperature':'1.0',
+
```

Comment:
again this logic can be moved to edly app methods

Reply:
done